cmd/ovn_leader_checker/ovn_leader_checker.go

```diff
@@ -0,0 +1,25 @@
+package ovn_leader_checker
+
+import (
+	"github.com/kubeovn/kube-ovn/pkg/ovn_leader_checker"
+	"os"
+)
+
+func CmdMain() {
+	cfg, err := ovn_leader_checker.ParseFlags()
+	if err != nil {
```

Comment:
It should run periodically no, not once.

We don't need to exit on error, log the error message is enough.

Reply:
i modified codes just like you said.